src/app/containers/CpsAsset/index.jsx

```diff
@@ -40,58 +38,72 @@ const SingleContentWrapper = styled.div`
   }
 `;
 
-const FeaturesAnalysis = ({ content }) => {
+const CpsAsset = ({
```

Comment:
should we call this `StoryPageOnwardJourney` or `CpsOnwardJourney` container instead ? I don't have a preference but I think `CpsAsset` container might imply this is meant for housing cps assets

Reply:
I like `CpsOnwardJourney` 😄